js/layout-javascript/news-feed-code.js

```diff
@@ -557,14 +557,17 @@ DynamicList.prototype.attachObservers = function() {
             {
               label: 'Delete',
               action: function (i) {
-                var options = {
+                Fliplet.Modal.confirm({
```

Comment:
We should only change to use `Fliplet.Modal` for Studio interface. This won't work in native apps.

Reply:
— as per note on developers documentation https://developers.fliplet.com/API/fliplet-core.html#modal

Reply:
@tonytlwu Got it. Removed fliplet modal.